commands/info/contributors.js

```diff
@@ -0,0 +1,38 @@
+const Discord = require("discord.js");
+const fetch = require("node-fetch");
+const config = require("../../config.json");
+
+module.exports = {
+  name: "contributors",
+  description: "List the contributors for Manami",
+  run: async (client, message, args) => {
+    var contributors = await (
+      await fetch("https://api.github.com/repos/SensionUltra/Manami/contributors", {
+        method: "GET",
+        headers: {
```

Comment:
There are no need for headers accourding to github 
https://docs.github.com/en/rest/reference/repos#list-repository-contributors
Try removing the headers